includes/blocks/example-block/block.json

```diff
@@ -9,5 +9,6 @@
 			"type" : "string"
 		}
 	},
-	"editorScript": "file:../../../dist/js/example-block.js"
-}
\ No newline at end of file
+	"editorScript": "file:../../../dist/js/example-block.js",
```

Comment:
@ryanwelcher 
I want to make it clear - the files referenced here are editor styles / scripts only.
Front end styles and scripts will be handled elsewhere in the theme. 
I feel using `index.css` is an appropriate pattern, but I'm wondering how to make it clear that it is editor only - or should we just assume engineers should "know that"? 

For additional clarity (from the FE minds eye in the scaffolds I've been on):
- The above files would be for 'editor overrides'
- Front end (team) styles for the block will be added in `assets/blocks/blockname.css`
- This file will be imported into `style.css` for the front end and `editor-style.css` for the back end (`postcss.config.js` adds scope to this file).

^^ are we suggesting changing this workflow? I'm fine if we are... just want to make sure we are all on the same page.

Reply:
@joesnellpdx this file contains the JavaScript that actually registers the block and is the dist version of the new entry point added for this block. This is part of the WordPress 5.5 block API changes.